app/docker/filters/filters.js

```diff
@@ -218,6 +218,22 @@ angular.module('portainer.docker')
     return runningTasks;
   };
 })
+.filter('runningcontainers', function () {
+  'use strict';
+  return function runningContainersFilter(containers) {
+    return containers.filter(function (container) {
+      return container.State === 'running';
+    }).length;
+  };
+})
+.filter('stoppedcontainers', function () {
+  'use strict';
+  return function stoppedContainersFilter(containers) {
+    return containers.filter(function (container) {
+      return container.State !== 'running';
```

Comment:
I think that we should use `container.State === 'stopped'` instead, otherwise containers in other state such as created, paused... will be considered stopped.

Reply:
so what state should these containers be? running?
I mean, if not, the count will not amount to total containers

Reply:
I think that we should simply not consider them in the total amount, we simply want the correct amount of running and stopped containers here.

Reply:
ok

Reply:
hmm, so I should also change the total amount to sum the running and stopped containers?

Reply:
I don't think so, we just need the list of all containers in the dashboard with details about running/stopped.

The user that reported the issue was just wondering why the number of running containers was not matching the number of running containers in the container list.